STYLE-GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Exercism's Style Guide
+
+This document acts as a Style Guide for the language and wording used in exercises.
+
+## General principle
+
+All content should be written in US english. 
+In the future other translations may occur, but the "official" Exercism natural langauge is US English.
+
+## Explain or substitude mathmatical and esotoric terms.
+
+In any place that mathematical terms are used they should be explained or substituted out for terms that require less domain knowledge.
+
+Examples:
+- Rather than using "natural numbers", we should use "positive integers" or "zero and positive integers". 
+- If we want to use the phrase "rational numbers", it must be explained in the introduction to the exercise.
+- Rather than using the word range (which can have different meanings in different contexts) use "greater than x and less than y".
+
+## Use consistency within an exercise.
+
+There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase"). 
```

Comment:
Perhaps this could be the first example of an agreed spelling?

Reply:
Sure. But I also want a rule that explains what happens when multiple valid spellings are used without a documented decision in this guide (ie, they should be exercise-consistent)

Reply:
lowercase seems to be more popular:
![image](https://user-images.githubusercontent.com/6978003/96163316-c0370e80-0ee7-11eb-8c17-3c844566476e.png)

https://english.stackexchange.com/questions/59409/lowercase-lower-case-or-lower-case

Reply:
Good enough for me 👍

Reply:
More popular != correct 😜

Reply:
I'd agree with lowercase, but I think this falls into the "it doesn't matter as long as its consistent within an exercise" bucket. Unless anyone feels strongly about this particular word, and wants to encode `lowercase` as a rule and find me another example for this bit :)